a3-live/src/components/TeamInterface.js

```diff
@@ -7,68 +7,88 @@ import FormControl from '@material-ui/core/FormControl';
 import ListItemText from '@material-ui/core/ListItemText';
 import Select from '@material-ui/core/Select';
 import Checkbox from '@material-ui/core/Checkbox';
+import {OutlinedInput} from "@material-ui/core";
+import { makeStyles } from '@material-ui/core/styles';
+import FilledInput from '@material-ui/core/FilledInput';
+import FormHelperText from '@material-ui/core/FormHelperText';
+import Card from "@material-ui/core/Card";
+import CardHeader from "@material-ui/core/CardHeader";
+import CardContent from "@material-ui/core/CardContent";
 
+const useStyles = makeStyles(theme => ({
+    root: {
+        display: 'flex',
+        flexWrap: 'wrap',
+    },
+
+    selectEmpty: {
+        marginTop: theme.spacing(2),
+    },
+    card: {
+        maxWidth: 345,
+    },
+}));
+/*
+const inputLabel = React.useRef(null); // TODO: Lookup in docs
+const [labelWidth, setLabelWidth] = React.useState(0);
+React.useEffect(() => {
+        setLabelWidth(inputLabel.current.offsetWidth);
+    }, []);
+*/
 
 function TeamInterface(props) {
-    const [TeamLead, setTeamLead] = React.useState({});
+    const classes = useStyles();
     const [TeamMembers, setTeamMembers] = React.useState([]);
+    const [TeamLead, setTeamLead] = React.useState(props.TeamLead);
     useEffect(() => {
         console.log('Effect: ', props);
     });
-    function handleChange(event) {
-        let {options} = event.target;
-        let value = [];
-        options.forEach(opt => {
-                if (opt.selected) {
-                    value.push(opt.value)
-                }
-            }
-        );
-        setTeamMembers(value);
+    function handleTeamLeadChange(event) {
+        setTeamLead(event.target.value)
+    }
+    function handleTeamMemberChange(event) {
+        setTeamMembers(event.target.value)
     }
-    function handleTeamMemberChange(event) {}
     function handleSave(event) {}
+    function EmployeeFullName(id) {
+        let foundEmployee = props.Employees.find(
+            function(emp) {return emp._id === id}
+        );
+        if (foundEmployee === undefined) return "undefined Employee. Check database";
+        return foundEmployee.FirstName + foundEmployee.LastName
+    }
     return (
         <div className="col-md-4">
-            <div className="panel panel-default">
-                <div className="panel-heading">
-                    <strong>{props.TeamName}</strong>
+            <Card className={classes.card}>
+                <CardHeader title="Here's TeamName">
                     <button className="btn btn-primary btn-xs pull-right" onClick={handleSave}>Save</button>
```

Comment:
Check out these M-UI [buttons](https://material-ui.com/components/buttons/)
```suggestion
                    <Button onClick={handleSave} variant="contained" color="primary" className={classes.button}>
        Save
      </Button>
```